test/lib/fastly_nsq/rake_task_test.rb

```diff
@@ -0,0 +1,64 @@
+require 'test_helper'
+require 'fastly_nsq/rake_task'
+
+describe MessageQueue::RakeTask do
+  before(:each) do
+    Rake::Task['begin_listening'].clear if Rake::Task.task_defined?('begin_listening')
+    allow_any_instance_of(MessageQueue::RakeTask).to receive(:output) { nil }
```

Comment:
Consider moving this stub into the tests themselves - it's the kind of thing that might surprise me if I didn't know about it. The line above should stay in this `before` block, though, since it's setup stuff.